integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -67,6 +77,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(AlluxioFuseFileSystem.class);
   private static final int MAX_OPEN_FILES = Integer.MAX_VALUE;
   private static final int MAX_OPEN_WAITTIME_MS = 5000;
+  private static final int MAX_NAME_LENGTH = 255;
```

Comment:
ideally, put some reference in comment here where the value `255` is from, similar to how the comment on `ID_NOT_SET_VALUE`